minari/storage/remotes/gcp.py

```diff
@@ -2,12 +2,18 @@
 from pathlib import Path
 from typing import Any, Optional
 
-from google.cloud import storage
-from tqdm.auto import tqdm
-
 from minari.storage.remotes.cloud_storage import CloudStorage
 
 
+try:
+    from google.cloud import storage
```

Comment:
`import google.cloud.storage as gcp_storage` should solve the pre-commit issue (and you need to change the storage reference accordingly)